Lib/test/test_curses.py

```diff
@@ -990,13 +986,15 @@ def test_color_attrs(self):
     @requires_curses_func('use_default_colors')
     @requires_colors
     def test_use_default_colors(self):
-        old = curses.pair_content(0)
+        # The current terminal fg/bg colors given by pair_content(0),
+        # which is backed by extended_pair_content(3), depends on the
+        # terminal emulator and its supported colors.
+        curses.pair_content(0)
```

Comment:
If we remove the check, the call to `curses.pair_content(0)` can be removed as well. It is no longer useful.

Reply:
I eventually decided to use the other PR. It's not really an issue and it could catch issues when we call `pair_content(0)` from the C code.